rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc

```diff
@@ -74,16 +90,12 @@ The latter might be the case when the bot has the status OK and a deployed proce
 
 When you restart the bot, the bot and all of its threads are initially stopped after the pending tasks have been completed. The subsequent restart is then performed in a clean environment, which means the problem should no longer occur.
 
-Restarting a bot will only work if that bot has the status OK. Otherwise it is placed in the task list and is executed as soon as the bot is OK again.
+Restarting a bot only works if that bot has the status OK. Otherwise RPA Manager places the restart in the task list and executes it soon as the bot is OK again.
```

Comment:
```suggestion
Restarting a bot only works if that bot has the status OK. Otherwise, RPA Manager places the restart in the task list and executes it as soon as the bot is OK again.
```